ImageAnalysis/image_analysis/analyzers/UC_GenericLightSpecCam.py

```diff
@@ -15,10 +15,16 @@
 if TYPE_CHECKING:
     from ..types import Array2D
 
+from pathlib import Path
+from scipy.interpolate import interp1d
+
 from ..base import LabviewImageAnalyzer
 from .online_analysis_modules import image_processing_funcs as process
 from .online_analysis_modules import photon_spectrometer_analyzer as analyze
 
+from ..utils import read_imaq_image, ROI
```

Comment:
This reminds me that we should test again if this old Python version on all the camera servers still complains about utils.